### DISCUSSION TITLE
When will be the next release

### DISCUSSION THREAD

**Discussion by ollimeier** [fonttools/fonttools#2543] [Q&A] [answered]
Hi there, 
I am wondering if there is already a schedule is for the next release of fonttools?

Thanks, 
Olli

**Comment by anthrotype** [comment, marked as answer]
sure, I can push one for you. Let me update the changelog first

**Reply by ollimeier** [reply]
Thanks :)

**Comment by anthrotype** [comment]
here:
https://github.com/fonttools/fonttools/releases/tag/4.30.0
https://pypi.org/project/fonttools/4.30.0/

give it a few minutes for PyPI to propagate the upload in case it doesn't install at first try

**Reply by Gwrgwer-Ofgqfrw** [reply]
https://stackoverflow.com/questions/76343468/automatically-adjust-font-metrics-to-match-a-different-font-using-fonttools
